tests/unit/store-actions.spec.ts

```diff
@@ -115,4 +115,58 @@ describe('testing actions', () => {
         expect(store.state.stateFiling).toHaveProperty('header')
       })
   })
+
+  it('fetches consent to continuation out state filing', async () => {
+    // arrange
+    const sampleStateFiling = {
+      business: {
+        foundingDate: '2022-12-06T19:13:16.000+00:00',
+        identifier: 'BC0871273',
+        legalName: 'AC BEN 2022.DEC.6 18.14 TEST CORP.',
+        legalType: 'BEN'
+      },
+      consentContinuationOut: {
+        expiry: '2023-12-31',
+        orderDetails: 'Ledger notes.',
+        order: {
+          fileNumber: 'F12345',
+          effectOfOrder: true
+        }
+      },
+      header: {
+        affectedFilings: [],
+        availableOnPaperOnly: false,
+        certifiedBy: 'BCREGTEST Delphia EIGHTEEN',
+        colinIds: [9179235],
+        comments: [],
+        date: '2023-01-14T01:07:00.445972+00:00',
+        deletionLocked: false,
+        documentOptionalEmail: 'completing-party@example.com',
+        effectiveDate: '2023-01-14T01:07:12.136444+00:00',
+        filingId: 139770,
+        inColinOnly: false,
+        isCorrected: false,
+        isCorrectionPending: false,
+        isFutureEffective: false,
+        name: 'dissolution',
+        paymentStatusCode: 'CREATED',
+        paymentToken: '25332',
+        status: 'COMPLETED',
+        submitter: 'bcsc/x5levvougen54gnvtvcyh7nrelqcyt4t'
+      }
+    }
+    // act
+    jest.spyOn(LegalServices, 'fetchFiling').mockImplementation(
+      (): Promise<any> => {
+        return Promise.resolve(sampleStateFiling)
+      }
+    )
+    await store.dispatch('retrieveStateFiling', 'http://localhost/')
+      .then(() => {
+        expect(LegalServices.fetchFiling).toHaveBeenCalled()
+        expect(store.state.stateFiling).toHaveProperty('business')
+        expect(store.state.stateFiling).toHaveProperty('consentContinuationOut')
+        expect(store.state.stateFiling).toHaveProperty('header')
+      })
+  })
```

Comment:
This doesn't do much but can be extended. (The current ticket is kind of a framework, not filing implementation, ticket.)